extensions/rust/tests/test_pymatgen_compat.py

```
@@ -1,1209 +1,652 @@
+
+    @pytest.fixture(scope="class")
+    def structures(self) -> dict[str, Structure]:
+        """Load matterviz structures."""
+        result = {}
+        for json_file in MATTERVIZ_STRUCTURES_DIR.glob("*.json"):
+            try:
+                data = json.loads(json_file.read_text())
+                if data.get("@class") == "Structure":
+                    result[json_file.stem] = Structure.from_dict(data)
+            except Exception:
+                pass
```

Ruff (0.14.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)